src/rules/enforce-callback-memo.ts

```
@@ -31,6 +31,89 @@ export default createRule<[], MessageIds>({
+      }
+
+      return false;
+    }
+
+    function getParentFunctionParams(node: TSESTree.Node): string[] {
+      let current: TSESTree.Node | undefined = node.parent;
+
+      while (current) {
+        if (isFunction(current)) {
+          const params: string[] = [];
+          current.params.forEach(param => {
```

ESLint

[error] Replace `param` with `(param)`

(prettier/prettier)